ui/src/components/DownloadButton/DownloadButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import Button from '@splunk/react-ui/Button';
+
+interface DownloadButtonProps {
+    // needs to be same domain if not it will just open link
+    fileUrl: string;
+    fileNameAfterDownload: string;
+    children: ReactElement | ReactElement[] | string;
+}
+
+function DownloadButton(props: DownloadButtonProps) {
+    return (
+        <Button
+            target="_blank"
+            to={props.fileUrl}
+            rel="noopener noreferrer"
```

Comment:
```suggestion
```
https://chromestatus.com/feature/6140064063029248

Reply:
yea, make sense, just for visibility it is supported by almost all browsers if not all (https://caniuse.com/mdn-html_elements_a_implicit_noopener)